index.css

```diff
@@ -78,8 +88,372 @@
     margin-left: 10%;
   }
 
+  .works-centered img {
+    width: 100%;
+  }
+
+  .attach-icon img {
+    float: left;
+    width: 5%;
+    height: 5%;
+    margin-right: 2%;
+  }
+
   .social a img {
     width: 19.99px;
     height: 19.96px;
   }
+
+  .extra {
+    margin-left: 6%;
+    margin-right: 6%;
+  }
```

Comment:
- [ ] Please use CSS GRID in the main container of the works and Flexbox for each card, it is a requirement 👇🏽 

> In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.